feature/bgp/tests/local_bgp_test/local_bgp_test.go

```diff
@@ -232,9 +232,16 @@ func TestDisconnect(t *testing.T) {
 	gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*120, oc.Bgp_Neighbor_SessionState_ESTABLISHED)
 	gnmi.Replace(t, ate, ateConfPath.Bgp().Neighbor(dutIP).Enabled().Config(), false)
 	gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*120, oc.Bgp_Neighbor_SessionState_ACTIVE)
-	code := gnmi.Get(t, dut, nbrPath.Messages().Received().LastNotificationErrorCode().State())
-	if code != oc.BgpTypes_BGP_ERROR_CODE_CEASE {
-		t.Errorf("On disconnect: expected error code %v, got %v", oc.BgpTypes_BGP_ERROR_CODE_CEASE, code)
+	code := gnmi.Lookup(t, dut, nbrPath.Messages().Received().LastNotificationErrorCode().State())
```

Comment:
If code is not present and deviation flag is not enable, should we emit an error?

Reply:
If code is not present, we handle the case using deviation flag where we display leaf not present, and we dont fail the case due to the deviation set.

If code is not present, and deviation not set, then the test would get failed due to error of the last notification error code leaf not present.

Reply:
Addressed.